src/PythonPlot.jl

```diff
@@ -190,7 +190,7 @@ Base.close(f::Figure) = plotclose(f)
 
 # rename to avoid type piracy:
 @doc LazyHelp(pyplot,"close") plotclose() = pyplot.close()
-plotclose(f::Figure) = pyconvert(Int, plotclose(pyconvert(Int, f.number)))
+plotclose(f::Figure) = pyconvert(Union{Nothing,Int}, plotclose(pyconvert(Int, f.number)))
```

Comment:
It looks like we can just close the figure object directly nowadays, rather than getting `f.number` first:
```suggestion
plotclose(f::Figure) = pyplot.close(f)
```

Reply:
Oh, but we track it in `withfig_fignums`.